script/Makefile

```diff
@@ -212,7 +212,8 @@ codegen-tools-install: controller-gen
 	@# We must force the installation to make sure we are using the correct version
 	@# Note: as there is no --version in the tools, we cannot rely on cached local versions
 	@echo "Installing k8s.io/code-generator tools with version $(CODEGEN_VERSION)"
-	go install k8s.io/code-generator/cmd/applyconfiguration-gen@$(CODEGEN_VERSION)
+	#go install k8s.io/code-generator/cmd/applyconfiguration-gen@$(CODEGEN_VERSION)
+	go install github.com/astefanutti/code-generator/cmd/applyconfiguration-gen@pr-144-install
```

Comment:
Do we already have a PR opened with those changes? My concern is that once we bump `CODEGEN_VERSION` we won't be aligned and potentially screw up things.

Reply:
I think @astefanutti has prepared a pr to fix this upstream

Reply:
Okey, please, let's track and make sure to follow up it.

Reply:
The fix upstream kubernetes/kubernetes#114920 will be picked up once we upgrade our dependency on code-generator.